src/main/resources/templates/db_scripts/schema.json

```diff
@@ -1067,6 +1067,11 @@
       "run": "after",
       "snippetPath": "addInstanceFormatsAudioBelt.sql",
       "fromModuleVersion": "20.3.0"
+    },
+    {
+      "run": "after",
+      "snippetPath": "dropLegacyItemEffectiveLocationFunctions.sql",
+      "fromModuleVersion": "21.0.0"
```

Comment:
Version 21.0.0 has already been released. I think that means that this won't be executed when an upgrade is done.

As the next planned release is 21.1.0, I think that is what the from module version should be.

Reply:
Fixed.